### ISSUE TITLE
High number of bacterial genes in phamb assembled bins

### ISSUE THREAD

**Issue by ShailNair** [RasmussenLab/phamb#47]
HI,

I used phamb with [recommended workflow(not in parallel) ](https://github.com/RasmussenLab/phamb#how-to-run---not-in-parallel---quick-and-dirty) with the default settings on my assembled metagenomic contigs (mixed of all microbial contigs). Later, I used CheckV ( with prodigal -m option enabled) on the concatenated fasta file. Strangely, CheckV analysis revealed that a large number of the bins contained a high number of host (bacterial) genes, accounting for more than 50% (many contigs with more than 70%) of the total number of genes. Surprisingly, CheckV indicates that many of these bins are complete and without contamination. However, the presence of such a large number of host genes will interfere in the downstream analysis. I have attached my checkv results for your reference. 
[quality_summary.txt](https://github.com/RasmussenLab/phamb/files/10458498/quality_summary.txt)

**Comment by joacjo** [comment]
Hi Shail 

The reason for running CheckV is to identify reliable viral-bins, therefore we recommend to only consider Medium and High-quality bins for further analysis (those based on the AAI-model). Low-quality shall always be considered with a lot of scepticism. 

As for "Complete-bins" I checked your quality file, which had 4 examples, I see in those that a lot of sequence has been removed by CheckV. Perhaps you wanna evaluate the resulting cleaned sequences by CheckV.

Best,
Joachim
